tests/integration/buildcmd/test_build_terraform_applications_other_cases.py

```diff
@@ -310,10 +310,10 @@ def test_invoke_function(self):
 
         _, stderr, return_code = self.run_command(build_cmd_list, env=environment_variables)
         LOG.info(stderr)
-        self.assertNotEqual(return_code, 0)
+        self.assertEqual(return_code, 0)
```

Comment:
Why did this assertion flip?

Reply:
It was a mistake, we need to make sure that the build is successful